Sources/Steepshot/Steepshot.Core/Models/Requests/TransferModel.cs

```diff
@@ -18,15 +18,18 @@ public class TransferModel : AuthorizedActiveModel
 
         public CurrencyType CurrencyType { get; set; }
 
+        public string ChainCurrency { get; set; }
+
         public string Memo { get; set; }
 
-        public TransferModel(string login, string activeKey, string recipient, long value, byte precussion, CurrencyType currencyType)
+        public TransferModel(string login, string activeKey, string recipient, long value, byte precussion, CurrencyType currencyType, string chainCurrency)
```

Comment:
CurrencyType currencyType == string chainCurrency  now Currency  in steem looks like @000000013 - do not use chainCurrency